app/views/posts/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render "shared/user"%> 
```

Comment:
- [Optional] Good Job I can see all the links as was requested by the last reviewer 
![image](https://user-images.githubusercontent.com/74744787/215219981-0c236d12-03d2-45e1-b244-5416c19bb655.png)
![image](https://user-images.githubusercontent.com/74744787/215220010-e2aa9bdf-cdcc-4b3c-851c-1b17c58c5e72.png)
![image](https://user-images.githubusercontent.com/74744787/215220029-d38ef513-58d8-487c-86e0-30003c5f43ea.png)
![image](https://user-images.githubusercontent.com/74744787/215220047-447d7443-af68-49b3-b184-c370b031ea37.png)
but I noticed that the post count isn't updated unless the user makes a comment on the post !! Please fix this 
![image](https://user-images.githubusercontent.com/74744787/215220517-e9a71afc-7561-41b5-8c91-cee70f415b6a.png)